docs/content/latest/deploy/multi-dc/2dc-deployment.md

```diff
@@ -24,126 +23,168 @@ showAsideToc: true
+## Load data into the producer universe
 
-Note that this time, “yugabyte-producer” will be set up to consume data from “yugabyte-consumer”.
+Once you've set up replication, do the following to load data into the producer universe:
 
-## 4. Load data into producer universe
+1. Download the YugabyteDB workload generator JAR file (`yb-sample-apps.jar`) from [GitHub](https://github.com/yugabyte/yb-sample-apps/releases).
 
-1. Download the YugabyteDB workload generator JAR file (`yb-sample-apps.jar`) from [GitHub](https://github.com/yugabyte/yb-sample-apps).
+1. Start loading data into “yugabyte-producer”.
 
-2. Start loading data into “yugabyte-producer” using the YugabyteDB workload generator JAR.
+    **YSQL example**
+
+    ```sh
+    java -jar yb-sample-apps.jar --workload SqlSecondaryIndex  --nodes 127.0.0.1:5433
+    ```
+
+    **YCQL example**
+
+    ```sh
+    java -jar yb-sample-apps.jar --workload CassandraBatchKeyValue --nodes 127.0.0.1:9042
+    ```
+
+1. For bidirectional replication, repeat the preceding step in the "yugabyte-consumer" universe.
+
+**Next**, continue to the next section to verify that replication is working correctly.
+
+## Verify replication
+
+**For unidirectional replication**, connect to “yugabyte-consumer” universe using the YSQL shell (`ysqlsh`) or the YCQL shell (`ycqlsh`), and confirm that you can see expected records.
+
+**For bidirectional replication**, repeat the previous step, but pump data into “yugabyte-consumer”. To avoid primary key conflict errors, keep the key space for the two universes separate.
+
+{{< tip title="Checking replication lag" >}}
+Replication lag is computed at the tablet level as:
 
-**YSQL example**
+`replication lag = hybrid_clock_time - last_read_hybrid_time`
+
+Where `hybrid_clock_time` is the hybrid clock timestamp on the producer's tablet-server, and `last_read_hybrid_time` is the hybrid clock timestamp of the latest transaction pulled from the producer.
+
+An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates replication lag for you. The script requires the [`jq`](https://stedolan.github.io/jq/) package.
+
+The following example generates a replication lag summary for all tables on a cluster. You can also request an individual table.
 
 ```sh
-java -jar yb-sample-apps.jar --workload SqlSecondaryIndex  --nodes 127.0.0.1:5433
+./determine_repl_latency.sh -m 10.150.255.114,10.150.255.115,10.150.255.113
 ```
 
-**YCQL example**
+Call `determine_repl_latency.sh -h` to get a summary of all command options.
+
+{{< /tip >}}
+
+**If you're using Yugabyte Platform to manage universes**, continue to the next section. If not, you're done!
+
+## Enable universes in Platform
+
+If you're using Yugabyte Platform to manage your universes, you need to call the following REST API endpoint on your Platform instance _for each universe_:
```

Comment:
@daniel-yb @polarweasel can you reword this to please? 

you need to call the following REST API endpoint on your Platform instance for each universe (producer and consumer) involved in 2DC replication